en/monitoring/discovery/installation.md

```diff
@@ -28,3 +28,40 @@ products and go to the **Administration > Extensions > Manager** menu.
 5. To get ready-to-use discovery rules, go to the **Configuration > Plugin
 Packs** page and [install the plugin packs](../pluginpacks.html#pack-installation) for the 
 discovery providers you want.
+
+
+## Setting up a dedicated autodiscovery account
+
+We recommend that you set up a dedicated technical account that will execute the discovery jobs in the background (different from the one you use to create and launch jobs in the interface). 
+
+1. On page **Configuration > Users > Contacts/Users**, create a dedicated user. On the **Centreon Authentication** tab, give it the following rights:
+    - **Reach Centreon front-end**: **No**
+    - **Admin**: **Yes**
+    - **Reach API Configuration**: **Yes**.
+
+2. In the Central server's terminal, edit the following file:
+
+    ```
+    /etc/centreon-gorgone/config.d/31-centreon-api.yaml
+    ```
+    Replace the default username and password by the credentials of your dedicated autodiscovery account.
+
+    Example:
+
+    ```
+    gorgone:
+        tpapi:
+        - name: centreonv2
+        base_url: "http://127.0.0.1/centreon/api/beta/"
+        username: autodisco
+        password: 27v3Ax5hffQRYV5x
```

Comment:
```suggestion
        password: XXXXXXXXXX
```

Reply:
It will be more obvious that the passwords must be changed if you write XXXXXXXXX instead of something that looks like a real password (a lot of people copy and paste things without thinking, I do it myself :'D)